com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.compute

```diff
@@ -457,6 +460,31 @@ VoxelLighting EvaluateVoxelLightingLocal(LightLoopContext context, uint groupIdx
     return lighting;
 }
 
+float3 EvaluateVoxelDiffuseGI(PositionInputs posInput, JitteredRay ray, float t0, float dt, float rndVal, float extinction)
+{
+#if SAMPLE_PROBE_VOLUMES
+    float tOffset, weight;
+    ImportanceSampleHomogeneousMedium(rndVal, extinction, dt, tOffset, weight);
+
+    float t = t0 + tOffset;
+    posInput.positionWS = ray.originWS + t * ray.jitterDirWS;
+
+    float3 apvDiffuseGI;
+    EvaluateAdaptiveProbeVolume(GetAbsolutePositionWS(posInput.positionWS), apvDiffuseGI);
```

Comment:
Random question: I guess EvaluateAdaptiveProbeVolume evaluate the lighting with the lambert consine convolution coefficient in it. We should remove it to apply on volumetric. Similar to what is done with Ambient probe in SetPreconvolvedAmbientLightProbe() function

Reply:
Good point